includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -384,6 +396,12 @@ public function filter_data_amp_attributes( $attributes, $amp_data ) {
 		if ( isset( $amp_data['noloading'] ) ) {
 			$attributes['data-amp-noloading'] = '';
 		}
+		if ( isset( $amp_data['lightbox'] ) ) {
+			$attributes['data-amp-lightbox'] = '';
+			$attributes['on']                = 'tap:' . self::AMP_IMAGE_LIGHTBOX_ID;
```

Comment:
Per above, I think this needs to auto-increment a number to append to `amp-image-lightbox-`.